pkg/monitoring/metrics/metrics_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package metrics
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestMetrics(t *testing.T) {
```

Comment:
Currently, `make unittest` will not run this test. You need to add the `./pkg/...` path to the makefile variables here:
https://github.com/kubevirt/ssp-operator/blob/4570a61c9fea3f11e4faa7ac01791bc7bd6bc198/Makefile#L56-L57

Reply:
updated